massa-bootstrap/src/listener.rs

```diff
@@ -43,6 +43,8 @@ impl BootstrapTcpListener {
         if addr.is_ipv6() {
             socket.set_only_v6(false)?;
         }
+        // This is needed for the mio-polling system, which depends on the socket being non-blocking.
```

Comment:
You mention in discord a more refined explination of the pathology around not setting non-blocking. If you feel that might be useful context, consider providing the extra detail.